frame/scored-pool/src/lib.rs

```diff
@@ -407,16 +407,20 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
 		new_members.sort();
 
 		let old_members = <Members<T, I>>::get();
+		// TODO: What to do in case of error?
+		let _ = T::MembershipChanged::ensure_can_change_members(&new_members, &old_members);
 		<Members<T, I>>::put(&new_members);
 
 		match notify {
 			ChangeReceiver::MembershipInitialized =>
 				T::MembershipInitialized::initialize_members(&new_members),
-			ChangeReceiver::MembershipChanged =>
-				T::MembershipChanged::set_members_sorted(
+			ChangeReceiver::MembershipChanged => {
+				// TODO: Is this safe?
+				let _ = T::MembershipChanged::set_members_sorted(
```

Comment:
```suggestion
				T::MembershipChanged::set_members_sorted(
```

Why are you doing this everywhere?

Reply:
I figured it would be good to have an indication of whether the change happened as intended even without calling `ensure_can_change_members` and thus return the number of new members.
So now rustc complains about expecting `()` and finding `usize` (as intended)  which leads to this :-)